automation/features/pageobjects/governance.page.js

```diff
@@ -0,0 +1,18 @@
+import Page from "./page";
+
+class GovernancePage extends Page {
+  get proposalStatesTestId() {
```

Comment:
Can these just be instance properties and not gettters?

Reply:
yes we can do that and i've been trying it but we'll need to refactor quiet a few pages to do all of them 
From :    get proposalStatesTestId() {
    return "governance-proposal-state";
  }

TO:  proposalStatesTestId =  "governance-proposal-state"

is there a big reason to not use getters ?